grpc/src/client/channel.rs

```diff
@@ -22,19 +22,119 @@
  *
  */
 
-use std::{any::Any, str::FromStr, sync::Arc};
+use core::panic;
+use std::{
```

Comment:
This import style differs from what we do in the rest of tonic, we should think about aligning that.

Reply:
I have no strong feelings on this - just let me know how to do it and I can try to fix everything up.